src/main/play-doc/operation/Install.md

```diff
@@ -165,45 +243,94 @@ Proxying application endpoints is required when running more than one instance o
 
 We will be using `HAProxy`. Install HAProxy version 1.5 or newer.
 
-``` bash
+```bash
 [172.17.0.1]$ sudo apt-get -y install haproxy
 ```
 or
-``` bash
+```bash
 [172.17.0.1]$ sudo yum install haproxy
 ```
 
 On Red Hat Enterprise Linux (RHEL) 6, haproxy is in the RHEL Server Load Balancer (v6 for 64-bit x86_64) rhel-lb-for-rhel-6-server-rpms channel. You'll need to add this channel to your server.
 
 On some Debian distributions you may need to add a dedicated Personal Package Archive (PPA) in order to install HAProxy 1.5 via the package manager. For example:
 
-``` bash
+```bash
 [172.17.0.1]$ sudo add-apt-repository -y ppa:vbernat/haproxy-1.5
 [172.17.0.1]$ sudo apt-get update
 [172.17.0.1]$ sudo apt-get -y install haproxy
 ```
 
 ConductR provides a ConductR-HAProxy bundle that listens for bundle events from ConductR and updates the local HAProxy configuration file accordingly. We must specifically allow the bundle to use `sudo` to reload HAProxy.
 
-First, we have the user `conductr` own the HAProxy config file.
+First, we have the user `conductr-agent` own the HAProxy config file.
 
-``` bash
-[172.17.0.1]$ sudo chown conductr:conductr /etc/haproxy/haproxy.cfg
+```bash
+[172.17.0.1]$ sudo chown conductr-agent:conductr-agent /etc/haproxy/haproxy.cfg
+```
+
+### Installing HAProxy reload script
+
+After updating the HAProxy configuration file, ConductR-HAProxy will signal HAProxy to reload for the updated configuration.
+
+Provide the following HAProxy reload script in `/usr/bin/reloadHAProxy.sh`. The reload script will determine whether `systemv`, `upstart`, or `systemd` is being used in the underlying operating system, and it will issue HAProxy reload accordingly.
+
+```bash
+#!/usr/bin/env bash
+
+if [[ `systemctl` =~ -\.mount ]]; then
+  INIT=systemd
+elif [[ `/sbin/init --version` =~ upstart ]]; then
+  INIT=upstart
+elif [[ -f /etc/init.d/cron && ! -h /etc/init.d/cron ]]; then
+  INIT=systemv
+else
+  INIT=unknown
+fi
+
+if [[ $INIT = upstart || $INIT = systemv ]]; then
+   /etc/init.d/haproxy reload
+else
+  /bin/systemctl reload  haproxy.service
+fi
```

Comment:
I thought we were doing away with this script and just provide one liners depending on the OS...